tests/nix_tests/ibc_utils.py

```diff
@@ -1,116 +1,118 @@
     wait_for_port(ports.grpc_port(evmos.base_port(0)))  # evmos grpc
 
-    version = {"fee_version": "ics29-1", "app_version": "ics20-1"}
-    incentivized_args = (
-        [
-            "--channel-version",
-            json.dumps(version),
-        ]
-        if incentivized
-        else []
-    )
-
-    # pystarport (used to start the setup), by default uses ethereum
-    # hd-path to create the relayers keys on hermes.
-    # If this is not needed (e.g. in Cosmos chains like Stride, Osmosis, etc.)
-    # then overwrite the relayer key
-    if "chainmain" not in other_chain_name:
-        subprocess.run(
-            [
-                "hermes",
-                "--config",
-                hermes.configpath,
-                "keys",
-                "add",
-                "--chain",
-                other_chain_name,
-                "--mnemonic-file",
-                evmos.base_dir.parent / "relayer.env",
-                "--overwrite",
-            ],
-            check=True,
-        )
+    # relayer
+    hermes = Hermes(evmos.base_dir.parent / "relayer.toml")
 
-    subprocess.check_call(
-        [
-            "hermes",
-            "--config",
-            hermes.configpath,
-            "create",
-            "channel",
-            "--a-port",
-            "transfer",
-            "--b-port",
-            "transfer",
-            "--a-chain",
-            "evmos_9000-1",
-            "--b-chain",
-            other_chain_name,
-            "--new-client-connection",
-            "--yes",
-        ]
-        + incentivized_args
-    )
+    chains_to_connect = ["evmos_9000-1"]
 
-    if incentivized:
-        # register fee payee
-        src_chain = evmos.cosmos_cli()
-        dst_chain = other_chain.cosmos_cli()
-        rsp = dst_chain.register_counterparty_payee(
-            "transfer",
-            "channel-0",
-            dst_chain.address("relayer"),
-            src_chain.address("signer1"),
-            from_="relayer",
-            fees=f"100000000{other_chain_denom}",
+    # set up the other chains to connect to evmos
+    for chain in other_chains_names:
+        meta = IBC_CHAINS_META[chain]
+        other_chain_name = meta["chain_name"]
+        chain_instance = CosmosChain(
+            evmos.base_dir.parent / other_chain_name, meta["bin"]
         )
-        assert rsp["code"] == 0, rsp["raw_log"]
+        # wait for grpc ready in other_chains
+        wait_for_port(ports.grpc_port(chain_instance.base_port(0)))
+
+        chains[chain] = chain_instance
+        chains_to_connect.append(other_chain_name)
+        # pystarport (used to start the setup), by default uses ethereum
+        # hd-path to create the relayers keys on hermes.
+        # If this is not needed (e.g. in Cosmos chains like Stride, Osmosis, etc.)
+        # then overwrite the relayer key
+        if "chainmain" not in other_chain_name:
+            subprocess.run(
```

Comment:
we are using `subprocess.run(..., check=True)` here and `subprocess.check_call(...)` in the next block. Looks like both do the same, can we also use `check_call` here instead?